templates/0_docs/content_strips.html

```diff
@@ -73,7 +73,7 @@ <h3>Webinars and case studies</h3>
               attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
               ) | safe
             }}
-            <h4 class="p-heading-icon__title">Datasheet</h4>
+            <h4>Datasheet</h4>
```

Comment:
Just wondering why in this page only `<h4>` is used here instead of `<p class="p-heading--4">`

Reply:
This page is a strange one as it's kind of secret to us, and it's not really a web page in a particular order that makes sense it's just a list of different examples. (I also noticed there was a `<h3>` before it so semantically it makes sense to have a `<h4>`)